colossalai/inference/modeling/policy/nopadding_baichuan.py

```diff
@@ -42,17 +44,38 @@ def module_policy(self):
             ]
         )
 
+        # used for Baichuan 13B
+        policy["BaichuanLayer"] = ModulePolicyDescription(
+            sub_module_replacement=[
+                SubModuleReplacementDescription(
+                    suffix="mlp",
+                    target_module=NopadBaichuanMLP,
+                ),
+                SubModuleReplacementDescription(
+                    suffix="self_attn",
+                    target_module=NopadBaichuanAttention,
+                ),
+            ]
+        )
+
         self.append_or_create_method_replacement(
             description={"forward": llama_causal_lm_forward}, policy=policy, target_key="BaichuanForCausalLM"
         )
         self.append_or_create_method_replacement(
             description={"forward": llama_model_forward}, policy=policy, target_key="BaichuanModel"
         )
+
+        # used for Baichuan 7B
         self.append_or_create_method_replacement(
             description={"forward": llama_decoder_layer_forward}, policy=policy, target_key="DecoderLayer"
         )
+        # used for Baichuan 13B
+        self.append_or_create_method_replacement(
+            description={"forward": llama_decoder_layer_forward}, policy=policy, target_key="BaichuanLayer"
```

Comment:
Same as above, though the number of lines keep unchanged.

Reply:
Done